rule.go

```diff
@@ -28,6 +28,12 @@ type Rule struct {
 	IPProto           int
 	UIDRange          *RuleUIDRange
 	Protocol          uint8
+
+	// Type is the unix.RTN_* rule type, such as RTN_UNICAST
+	// or RTN_UNREACHABLE.
+	// When adding a new rule, zero means automatic.
+	// Only supported on Linux.
```

Comment:
Why does this element need all this explanation while all other ones don't?

Reply:
I'm happy to remove it if you prefer, it just seemed like useful documentation for any consumer of this new field.

Reply:
Yes let's remove it, unless it deviates from iproute2.
Assumption is who uses this library is familiar with iproute2 and can always look for reference uses there and check kernek code as well. Also you are adding UT which shows an example, which is great.
Thank you for your contribution.

Reply:
Comment removed.